### ISSUE TITLE
cuckoo error in centos7

### ISSUE THREAD

**Issue by chasetsai** [cuckoosandbox/cuckoo#1085]
hello!
I install cuckoo in centos7.I can open cuckoo,web and db. It's an error occurred when I submit a sample in web : 
ConnectionError: HTTPConnectionPool(host='192.168.56.101%0asnapshot%20=%20win7_0901', port=8000): Max retries exceeded with url: / (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x2501f90>: Failed to establish a new connection: [Errno -2] Name or service not known',))

bellow is my config
cuckoo.conf
[cuckoo.conf.txt](https://github.com/cuckoosandbox/cuckoo/files/467082/cuckoo.conf.txt)

virtualbox.conf
[virtualbox.conf.txt](https://github.com/cuckoosandbox/cuckoo/files/467084/virtualbox.conf.txt)

how can I fix this error??
thank you all:))

**Comment by jbremer** [comment]
Hi, sorry for the late reply. The issue is (apparently - I wasn't aware of this behavior) that because of the space in front of your `snapshot =` line, that `ConfigParser` (the awful Python library that we use) returns odd data:
```python
In [166]: import ConfigParser
In [176]: a=ConfigParser.ConfigParser()
In [169]: a.readfp(io.BytesIO("[a]\nb = c\nd = e")
In [173]: a.items("a")
Out[173]: [('b', 'c'), ('d', 'e')]

In [176]: a=ConfigParser.ConfigParser()
In [177]: a.readfp(io.BytesIO("[a]\nb = c\n d = e"))
In [178]: a.items("a")
Out[178]: [('b', 'c\nd = e')]
```

I'm going to think about a workaround (or at least identification) for this issue in #863.

**Comment by jbremer** [comment]
This has now been implemented as part of #863, I'll be pushing the related code shortly. Closing this issue as resolved.
